src/writer/async_tokio.rs

```diff
@@ -57,6 +60,58 @@ impl<W: AsyncWrite + Unpin> Writer<W> {
         Ok(())
     }
 
+    // /// Provides a simple, high-level async API for writing XML elements.
```

Comment:
Just remove the whole commented block. You always can restore the previous version by checkouting a commit that GitHub shows in a PR